Source/SubnetInjection/ValidateVnetLocationForEnterprisePolicy.ps1

```diff
@@ -11,7 +11,7 @@ $supportedVnetLocations = New-Object "System.Collections.Generic.Dictionary[[Str
 $supportedVnetLocations.Add("centraluseuap", "eastus|westus")
 $supportedVnetLocations.Add("eastus2euap", "eastus|westus")
 $supportedVnetLocations.Add("unitedstateseuap", "eastus|westus")
-$supportedVnetLocations.Add("unitedstates", "eastus|westus;centralus|eastus2")
+$supportedVnetLocations.Add("unitedstates", "eastus|westus;centralus|eastus2;usgovtexas|usgovvirginia")
```

Comment:
this change is necessary elsewhere as well. do we really call the usgovhigh region unitedstates as well?